data/implementations/index.json

```
@@ -536,18 +1373,26 @@
         "credentialSubject": {},
         "proof": {
           "type": "JsonWebSignature2020",
-          "created": "2021-10-02T18:02:47Z",
+          "created": "2021-11-06T17:39:22Z",
           "verificationMethod": "did:example:123#key-1",
           "proofPurpose": "assertionMethod",
-          "jws": "eyJhbGciOiJFUzI1NksiLCJiNjQiOmZhbHNlLCJjcml0IjpbImI2NCJdfQ..dPgV7u6_XDPtps4SUjwHmDpebBagiNlarUWuyB-DQaQFckYJLxUPX78_G3QBiI6NJsdC3mherg0usQ2RX3LgxQ"
+          "jws": "eyJhbGciOiJFUzI1NksiLCJiNjQiOmZhbHNlLCJjcml0IjpbImI2NCJdfQ..q3-f6-13wvXjBCZGrJNYbXb3kRHsTTwz7Xo3XEEK_fhxCt2w1EiDJE6mrkAO4bhNAcwhw3XELC9LZ7rnZC8fPQ"
         }
       },
       "verification": {
         "verified": true
       }
     },
-    "credential-0--key-2-secp256r1.json": {
-      "vector": {
+    "credential-0--key-2-secp256r1.vc-jwt.json": {
+      "vector": "vc-jwt",
+      "vectorContent": "eyJhbGciOiJFUzI1NiIsImtpZCI6ImRpZDpleGFtcGxlOjEyMyNrZXktMiJ9.eyJpc3MiOiJkaWQ6ZXhhbXBsZToxMjMiLCJzdWIiOnt9LCJ2YyI6eyJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscy92MSIsImh0dHBzOi8vdzNpZC5vcmcvc2VjdXJpdHkvc3VpdGVzL2p3cy0yMDIwL3YxIl0sInR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiXSwiaXNzdWVyIjoiZGlkOmV4YW1wbGU6MTIzIiwiaXNzdWFuY2VEYXRlIjoiMjAyMS0wMS0wMVQxOToyMzoyNFoiLCJjcmVkZW50aWFsU3ViamVjdCI6e319LCJuYmYiOjE2MDk1MjkwMDR9.ZXlKaGJHY2lPaUpGVXpJMU5pSXNJbUkyTkNJNlptRnNjMlVzSW1OeWFYUWlPbHNpWWpZMElsMTkuLlpwUUVCSVRFV2ppbmhkSl9IcDVtNUFRWllpaXhDbEdNZEYwMHdhVjlFeDFjSHlJdnR4d1NaR1FoTXR4TXFKSzd1bXVuNjBkWDZjTE9MNjZiLW5ZWGhR",
```

Same as https://github.com/decentralized-identity/JWS-Test-Suite/pull/24#discussion_r744316497
```suggestion
      "vectorContent": "eyJhbGciOiJFUzI1NiIsImtpZCI6ImRpZDpleGFtcGxlOjEyMyNrZXktMiJ9.eyJpc3MiOiJkaWQ6ZXhhbXBsZToxMjMiLCJzdWIiOnt9LCJ2YyI6eyJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscy92MSIsImh0dHBzOi8vdzNpZC5vcmcvc2VjdXJpdHkvc3VpdGVzL2p3cy0yMDIwL3YxIl0sInR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiXSwiaXNzdWVyIjoiZGlkOmV4YW1wbGU6MTIzIiwiaXNzdWFuY2VEYXRlIjoiMjAyMS0wMS0wMVQxOToyMzoyNFoiLCJjcmVkZW50aWFsU3ViamVjdCI6e319LCJuYmYiOjE2MDk1MjkwMDR9.0SqE4B-kpfsRY4k5YwYZg1YrY1V-zTYXrDnprb9WKo3Nzt5InQHyoLIcPON12EpiukydjNrELyMKiuicoJhIHA",
```